frontend/controller/actions/group.js

```diff
@@ -224,6 +241,13 @@ export default (sbp('sbp/selectors/register', {
     sbp('state/vuex/commit', 'setCurrentGroupId', groupId)
   },
   'gi.actions/group/addChatRoom': async function (params: GIActionParams) {
+    const contractState = sbp('state/vuex/state')[params.contractID]
+    for (const contractId in contractState.chatRooms) {
+      if (params.data.attributes.name.toUpperCase() === contractState.chatRooms[contractId].name.toUpperCase()) {
+        throw new GIErrorUIRuntimeError(L('Duplicated channel name'))
```

Comment:
`'Duplicated channel name'` => `'Duplicate chatroom name'`

Reply:
What about in the UI? When creating new channel?

Reply:
*Yes, sorry, meant `channel`